bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -324,10 +330,11 @@ def forward(self, x: torch.tensor) -> torch.tensor:
 class nucleus( torch.nn.Module ):
     """ Nucleus class which holds the validator model.
     """
-    def __init__( self, config, device ):
+    def __init__( self, config, device, subtensor ):
         super(nucleus, self).__init__()
         self.config = config
         self.device = device
+        self.max_n = subtensor.max_n 
```

Comment:
have we tested on nobunaga with a changing max_n value to make sure a changing size in the validator would not break anything?

Reply:
also, do we have to change the n in metagraph as well? or it dosent matter